org.jacoco.core.test/src/org/jacoco/core/test/validation/targets/Target01.java

```diff
@@ -173,40 +246,41 @@ public static void main(String[] args) {
 			break;
 		}
 
-		// 18. Break statement
+	}
+
+	private static void breakStatement() {
+
 		while (true) {
 			if (t()) {
 				break; // $line-executedbreak$
 			}
 			nop(); // $line-missedafterbreak$
 		}
 
-		// 19. Continue statement
+	}
+
+	private static void continueStatement() {
+
 		for (int j = 0; j < 1; j++) {
 			if (t()) {
 				continue; // $line-executedcontinue$
 			}
 			nop(); // $line-missedaftercontinue$
 		}
 
-		runReturn();
-		runImplicitReturn();
-
 	}
 
-	private static void runReturn() {
+	private static void returnStatement() {
 
-		// 20. Return statement
 		if (t()) {
 			return; // $line-return$
 		}
 		nop(); // $line-afterreturn$
 
 	}
 
-	private static void runImplicitReturn() {
+	private static void implicitReturn() {
 
-		// 21. Implicit return
 	} // $line-implicitreturn$
```

Comment:
@Godin Correct, every target method without an return statement could serve as a test target. Would prefer to keep aspects in different methods though.

What is actually missing is a test case for a explicit unconditional return. In this case there should be no code assigned to the closing brace. Will add this.